content/docs/atom-guide/dev-guide/shaders/azsl/_index.md

```diff
@@ -382,33 +382,33 @@ An important remark about partial SRG blocks is that when the compiler, **AZSLc*
         OUT.m_color = PassSrg::GetColor();
         return OUT;
     }
-  
-## <a name="rootconstant"></a>Root Constants
+```
+## Root Constants
 AZSL supports the definition of shader constants that conform to the DX12 concept of Root Constants: https://docs.microsoft.com/en-us/windows/win32/direct3d12/using-constants-directly-in-the-root-signature.  
   
 Root Constants, declared with the `rootconstant` keyword, will be transparently embedded into a `ConstantBuffer`. They are useful to define shader constants of frequent access. Because they are located in the root signature they can be read instantly without requiring additional levels of indirection.  
   
 Shader constants qualified as `rootconstant` are always declared globally. Examples:  
-  
+```cpp
     rootconstant float4x4 s_objectMatrix;
     rootconstant uint s_materialIndex;
-  
+```
 Root Constants should be used sparingly because the space to define root constants, the Root Signature, is shared with Root Descriptors and Descriptor Tables. In D3D12, the Root Signature is limited at 64 DWords (256 Bytes) worth of space that is shared among Root Constants, Root Descriptors and Descriptor Tables.  
   
-## <a name="option"></a>Shader Variant Options
+## <a name="option"></a>Shader variant options
 
-Before describing how to use Shader Variant Options in a shader, you should read this: [AZSL: Design Principles And Purpose Of Shader Variant Options.](shader-variant-options)  
+Before describing how to use Shader variant options in a shader, you should read this: [AZSL: Design Principles And Purpose Of Shader variant options.](shader-variant-options)  
```

Comment:
Is it necessary for users to read **AZSL: Design Principles And Purpose Of Shader variant options** before reading this section? 

This is the "overview" page of the AZSL docs, so I believe that users should get a glimpse of what shader variant options are without needing to read the other docs. Instead of requiring them to read the other docs first, I suggest briefly defining shader variant options in this section. Then, refer them to the other docs for more information. 
```suggestion

```

Reply:
Fixed.